Adiciona control para semántica

santiagobernalg opened the pull request:
Se adiciona radio para tipo de semántica y campo de texto para la semántica

santiagobernalg committed:
Adiciona control para semántica

Se adiciona radio para tipo de semántica y campo de texto para la semántica

davidherney reviewed (changes requested):
- Quitar la copia del jQuery
- Controlar los input con label / for
- Hacer más grande la modal
- El botón de la modal debe tener los estilos que estamos usando

santiagobernalg committed:
corrección de input, tamaño de modal, jQuery y botón

Se quita la copia del jQuery
Se corrigen input con label / for
Se hacer más grande la modal
Se ajustan estilos del botón de la modal
